### DISCUSSION TITLE
Per-user daemon on ondemand worker?

### DISCUSSION THREAD

**Discussion by ltalirz** [Azure/az-hop#976] [Q&A]
We have an application that requires users to execute long-running processes (daemons) that continue running after they've logged out from their session. These daemons communicate with the cyclecloud.

Would it be possible to install those on the OnDemand VM or is that not possible / not a good idea?

Mentioning @matt-chan for info

**Comment by xpillons** [comment]
it all depends on what these daemons does, but usually we don't expect users dameons to run on the ondemand portal for security and scalability reasons. one way would be to have these into long running jobs, so this would be fully dynamic. Or you can spin up a node manually from the CycleCloud UI and have these daemons started from a ssh user session .
